gcp_variant_transforms/libs/annotation/vep/vep_runner.py

```diff
@@ -85,6 +85,12 @@ def create_runner_and_update_args(known_args, pipeline_args):
     known_args.annotation_fields.append(known_args.vep_info_field)
   else:
     known_args.annotation_fields = [known_args.vep_info_field]
+  # TODO(bashir2): The VEP runner by default runs VEP with --allele_number hence
+  # we turn on this feature here. However, this might be inconsistent with other
+  # annotation fields that are originally present in input files, if they do not
+  # have ALLELE_NUM annotation. The fix is to make annotation ALT matching
+  # smarter to fall back on other matching methods if ALLELE_NUM is not present.
+  known_args.use_allele_num = True
```

Comment:
why not set the default value to true in the flag instead? setting this to true here essentially makes the flag a no-op?

Reply:
Hmm, why is setting it here makes the flag no-op? The reason the default is False is because we want the user be specific in telling us to use ALLELE_NUM; think about pre-annotated VCF scenarios (we discussed this when this feature was added). Here though is where we are running VEP ourselves and we run it with `--allele_number` hence setting the flag.

Once we implement the suggestion in the above TODO, maybe we can actually remove this flag and always look for ALLELE_NUM first. I added an extra note to the end of TODO accordingly but please let me know if I misunderstood your point.

Reply:
ah i see...that makes sense! thanks!